src/main/java/com/google/sps/travelbud/CityServlet.java

```diff
@@ -0,0 +1,93 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.travelbud;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.*;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+// hard coded data
+class City {
+  int id;
+  String name;
+  int country_ID;
+  String description;
+  List<Integer> event_ID; // putting event id for now, but will use the event entity later
+
+  City(int id, String name, int country_ID, String description, List<Integer> event_ID) {
+    this.id = id;
+    this.name = name;
+    this.country_ID = country_ID;
+    this.description = description;
+    this.event_ID = event_ID;
+  }
+}
+/** Servlet that returns some example content. */
+@WebServlet(urlPatterns = {"/api/cities/*"})
+public class CityServlet extends HttpServlet {
+  @Override
+
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("application/json;");
+
+    // set up events for each city
+    List<Integer> City1_events = new ArrayList<Integer>();
+    City1_events.add(1);
+
+    List<Integer> City2_events = new ArrayList<Integer>();
+    City2_events.add(2);
+
+    // list of cities
+    List<City> cities = new ArrayList<City>();
+    City c1 = new City(1, "Copenhagen", 1, "Some cool copenhagen description", City1_events);
+    City c2 = new City(2, "Busan", 2, "Some cool Busan description", City2_events);
```

Comment:
I would maybe refactor these as class constants. E.g.

```java
  private static Event CITY_1_EVENT_1 = new Event(...);
  private static Event CITY_2_EVENT_1 = new Event(...);
  private static Event CITY_2_EVENT_2 = new Event(...);
  private static City CITY_1 = new City(1, "Copenhagen", 1, "Some cool copenhagen description", Arrays.asList(CITY_1_EVENT_1));
  private static City CITY_2 = new City(2, "Busan", 2, "Some cool Busan description", Arrays.asList(CITY_2_EVENT_1, CITY_2_EVENT_2));
  private static List<City> CITIES = Arrays.asList(CITY_1, CITY_2);
```

Alternatively, for better readability you might even want to add those hardcoded example countries, events, and cities into the class files for the different types using the actual names of them. E.g. in `Country.java`:
```java
public class Country {
  static Country DENMARK = new Country(1, "Denmark", ...);
  static Country SOUTH_KOREA = new Country(1, "South Korea", ...);
  static List<Country> COUNTRIES = Arrays.asList(DENMARK, SOUTH_KOREA);
}
```
in `City.java`:
```java
public class City {
  static City COPENHAGEN = new City(1, "Copenhagen", Country.DENMARK.id, ...);
  static City BUSAN = new City(2, "Busan", Country.SOUTH_KOREA.id, ...);
  static List<City> CITIES = Arrays.asList(COPENHAGEN, BUSAN);
}
```
and so forth (maybe for the `Event` ones `COPENHAGEN_EVENT_1`, etc.?)